dask/array/core.py

```diff
@@ -1220,6 +1228,11 @@ def modf(x):
 modf.__doc__ = np.modf
 
 
+@wraps(np.around)
+def around(x, decimals=0):
+    return map_blocks(x, partial(np.around, decimals=decimals), dtype=x.dtype)
```

Comment:
Just didn't seem like there was any need to use elemwise and invoke the overhead of broadcasting, etc, given that around is a unary operation -- note that the decimals argument needs to be an int, not an array.

On Mon, Mar 30, 2015 at 7:45 AM, Matthew Rocklin <notifications@github.com>
wrote:

>> @@ -1220,6 +1228,11 @@ def modf(x):
>>  modf.__doc__ = np.modf
>>  
>>  
>> +@wraps(np.around)
>> +def around(x, decimals=0):
>> +    return map_blocks(x, partial(np.around, decimals=decimals), dtype=x.dtype)
> Or rather, now that I look below and see that you know about elemwise, why didn't elemwise work here?
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/ContinuumIO/dask/pull/112/files#r27396107